docs/how-to/migrate-database.md

```diff
@@ -95,7 +95,7 @@ Teku creates a LevelDB2 database, and starts from the specified recent state. Te
 <!-- links -->
 
 [Download the latest finalized state]: https://consensys.github.io/teku/#operation/getEthV1DebugBeaconStatesWithState_id
-[prune mode]: ../reference/cli/index.md#data-storage-mode
+[`prune` mode]: ../reference/cli/index.md#data-storage-mode
 [archive mode]: ../reference/cli/index.md#data-storage-mode
```

Comment:
if we're altering formatting of prune, should probably modify here too for archive.... minimal is another mode, but its default now... could potentially list...

Reply:
@rolfyone does minimal need to be included in this doc? Where? We do not mention it in the procedural content at all.

Reply:
at the very least its outside the scope of this change, just the formatting on archive will suffice.